portfolio/src/main/webapp/style.css

```diff
@@ -73,4 +73,12 @@ a:visited {
   margin: 25px;
   max-height: 300px;
   max-width: 300px;
+}
+
+.button {
+    background-color: #70b180;
+}
+
+.footer {
+  padding: 100px;
 }
```

Comment:
Please add a new line at the end of the file.

Reply:
Will do, thank you